Brocfile.js

```diff
@@ -17,8 +17,8 @@ var app = new EmberAddon();
 // please specify an object with the list of modules as keys
 // along with the exports of each module as its value.
 
-app.import('bower_components/jquery-ui/jquery-ui.js');
-app.import('bower_components/jquery-ui/ui/slider.js');
-app.import('bower_components/jquery-ui/themes/base/slider.css');
+app.import('bower_components/jquery-ui/ui/minified/jquery-ui.custom.min.js');
```

Comment:
@jasonmit , sorry for the delay,  I was doing that just on a special custom branch for my project. Talked with @yinaw, we did not want this change to impact the master code and will figure out a right solution later on master.